weaver/core/identity-management/iin-agent/package.json

```diff
@@ -19,8 +19,8 @@
     "author": "",
     "license": "ISC",
     "dependencies": {
-        "@hyperledger-labs/weaver-protos-js": "~1.5.0",
-        "@hyperledger-labs/weaver-fabric-interop-sdk": "~1.5.0",
+        "@hyperledger/cacti-weaver-protos-js": "file:./protos-js",
+        "@hyperledger/cacti-weaver-sdk-fabric": "file:./cacti-weaver-sdk-fabric",
```

Comment:
Is this a temporary change?

Reply:
oh sorry may be during build the `local` file replaced original one.